### PR TITLE
[pdf2md] add code for pdf2md demo

### PR DESCRIPTION
**Pull request by vincentsong-sj**
This demo includes three pdf parsers: LlmaSherpa, Unstructured, LlamaParse.

### CODE REVIEW

File: pdf2md/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Running PDF Parsers on Docker Containers
+## Overview
+This demo showcases running different PDF parsers on the same docker container. Three
```
Comment (pengfeng) [inline pdf2md/README.md]: "in" the same

File: pdf2md/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+2. **Docker**: It's required to have `docker` installed on your system. Specifically, we have tested this demo with Docker Engine Community version 25.0.1 on Linux. 
+
+3. **OpenAI API Key for ChatGPT**: If you wish to use the ChatGPT functionality within this demo, an OpenAI API key is required. Please note that usage of this API is subject to OpenAI's pricing and usage policies. We use OpenAI text generation models to optimize the parsing of some special components like titles or tables etc. Without this API key, you can still try all three approaches.
+   
+4. **LlamaParse API Key**: If you wish to try the newly launched LlamaParse API service, you need to get an API key from [here](https://cloud.llamaindex.ai/). As of today (02/26/2024), this API service is in preview mode for free. Without this API key, you will not be able to try LlamaParse.
+
+## Quick Start
+### Setup environment on AWS
+Please follow this [README file](../env-setup/aws/ubuntu-22.04/README.md) to setup the demo environment on AWS EC2. Note that, GPU is not needed in this demo, so you can run it on any instance that is installed Docker.
```
Comment (pengfeng) [inline pdf2md/README.md]: GPU is not "required" ...

on any instance with Docker.

**Review by pengfeng — APPROVED** [review]
A few minor issues. LGTM.

File: pdf2md/README.md (hunk 1 above)
Comment (vincentsong-sj) [reply pdf2md/README.md]: Use "in".

File: pdf2md/README.md (hunk 2 above)
Comment (vincentsong-sj) [reply pdf2md/README.md]: Use "not required"

**Pull request merged**